### ISSUE TITLE
id property

### ISSUE THREAD

**Issue by alexandrtovmach** [react-figma/figma-api-stub#7]
Does any way to generate or set manually an `id`?

**Comment by ilyalesik** [comment]
Hi! Right now it's possible to set `id` manually:

```typescript
const rect = figma.createRectangle();
// @ts-ignore
rect.id = "14:56";
```

But it's just workaround and `id` field is a part of Figma behavior, so I think `id` generating inside `figma-api-stub` should be implemented.

I don't know how exactly Figma aliased `id` field, maybe we can just generate string through [nanoid](https://github.com/ai/nanoid). PR is welcome. 🙌

**Comment by ilyalesik** [comment]
I've implemented simple `id` generating:
* `figma.root` got `0:0` 
* `figma.currentPage` got `0:1`
* Other nodes get `1:{n}`, where `n` started from `2` and increased after every node creation. E.g. this code
```javascript
const rect1 = figma.createRectangle();
const rect2 = figma.createRectangle();
console.log(rect1.id, rect2.id);
```
prints `1:2 1:3`.

It's available in latest version (`figma-api-stub@0.0.34`).

**Comment by alexandrtovmach** [comment]
@ilyalesik amazing, thanks
